src/semidiscretization.jl

```diff
@@ -72,6 +72,11 @@ end
 
 Compute the discrete coefficients of the continuous function `func` at time `t`
 associated with the semidiscretization `semi`.
+For example, the discrete coefficients of `func` for a discontinuous Galerkin
+spectral element method ([`DGSEM`](@ref)) are the values of `func` at the
+Lobatto-Legendre nodes. Similarly, a classical finite difference method will use
+the values of `func` at the nodes of the grid assoociated with the semidiscretization
+`semi`.
```

Comment:
Great comment - even a student not familiar with the DG method will understand that :+1:

Reply:
Thanks!